### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by chienggc** [django-wiki/django-wiki#1237] [Q&A] [answered]
python 3.10, django 4.0
after done registration , perform login, it redirect me to home page
but the top-left navigation bar i can still see login and signup button instead of profile

**Comment by benjaoming** [comment]
Do you have all the default middleware? See: https://docs.djangoproject.com/en/4.0/topics/http/middleware/#activating-middleware

**Reply by chienggc** [reply]
Yes, I have it under setting.py
![image](https://user-images.githubusercontent.com/103575743/207907811-6602ef06-095b-40ab-93b8-0f6edcffaf68.png)

**Reply by benjaoming** [reply]
Did this work previously or is it a new installation?

Does your `/admin` login work?

**Reply by chienggc** [reply]
new installation , admin doesn't work also

**Reply by benjaoming** [reply]
I think your problem is something about cookies and sessions, maybe your browser blocks it.

**Reply by oscarmcm** [reply]
Hi @chienggc do you have any of these settings? https://docs.djangoproject.com/en/4.1/topics/http/sessions/#settings

**Reply by oscarmcm** [reply]
And make sure you cookie session when working locally is not enabled for secure mode only
https://github.com/django-wiki/django-wiki/blob/main/testproject/testproject/settings/dev.py#L13-L20

**Reply by oscarmcm** [reply]
In the other hand, from your screenshot it seems that you're using the test project in Django Wiki, make sure you've executed the migrations and as well if you're running the app from another local domain that is not `localhost` you might need to update the `ALLOWED_HOSTS` and the `SESSION_COOKIE_DOMAIN `

I just checked the test app and seems to be working fine.

![image](https://user-images.githubusercontent.com/2553459/207918271-0903ea2b-7621-49f8-b769-0459d1083e58.png)

**Reply by chienggc** [reply]
i believe i might missed out something, i start again my installation on another computer with below steps, still cannot
below is my steps, kindly advise if i my steps are wrong @oscarmcm  @benjaoming 
1. download latest django-wiki test app on github and unzip it
2. pip install django
3. pip install django-markdown
4. pip install django-mptt
5. pip install django-sekizai
6. pip install sorl-thumbnail
7. pip install pillow
8. pip install django-nyt
9. pip install wiki
10. cd to the directory
11. run python manage.py migrate
12. run python manage.py runserver
13. open browser to register and login

**Comment by oscarmcm** [comment, marked as answer]
Hi @chienggc

So after checking your steps, I believe that doing `pip install wiki` (step 9) and running the downloaded code in step 11 is not quite right.

When doing step 9 this will try to install the latest PyPI version of Django Wiki and if I’m not wrong, that one doesn’t contain some changes for the local development.

I advise you to do `pip install . ` in the step 9 which will install the latest downloaded code.

On the other hand, from which directory are u running the app? and under which domain?
